src/packages/@app/pages/Collections/CollectionPage.ViewModel.ts

```diff
@@ -2384,6 +2387,28 @@ export default class CollectionsViewModel {
     return response;
   };
 
+  /**
+   * Fetches the collection guide document.
+   *
+   * @returns {Promise<Object>} - A promise that resolves to the collection guide document.
+   */
+  public fetchCollectionGuide = async (query) => {
+    const data = await this.guideRepository.findOne(query);
+    return data;
+  };
+
+  /**
+   * Updates the collection guide document's active status.
+   *
+   * @param {boolean} isActive - The new active status to set for the collection guide.
+   * @returns {Promise<void>} - A promise that resolves when the update is complete.
```

Comment:
no need to give data type in js doc curly braces, as we already using typescript